src/tests/dotnet-stack/DotnetStack.UnitTests.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>$(UnitTestTargetFrameworks)</TargetFrameworks>
```

Comment:
This one shells out - no need to do multitargeting (it would run the same tests n times)